src/api/common_api.js

```diff
@@ -252,41 +252,66 @@ module.exports = {
             }
         },
 
+        bucket_policy_principal: {
+            anyOf: [{
+                    type: 'string',
+                    enum: ['*']
+                }, {
```

Comment:
writing it here but it's not related to this file, updating nsfs tool is needed since we use bucket policy there and it's breaking the tool